cloud/azure/management/applicationinsights/mode/requests.pm

```diff
@@ -0,0 +1,193 @@
+#
+# Copyright 2021 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::management::applicationinsights::mode::requests;
+
+use base qw(cloud::azure::custom::mode);
+
+use strict;
+use warnings;
+
+sub get_metrics_mapping {
+    my ($self, %options) = @_;
+
+    my $metrics_mapping = {
+        'requests/count' => {
+            'output' => 'Server requests',
+            'label'  => 'requests-count',
+            'nlabel' => 'appinsights.requests.total.count',
+            'unit'   => '',
+            'min'    => '0',
+            'max'    => ''
+        },
+        'requests/duration' => {
+            'output' => 'Server response time',
+            'label'  => 'requests-duration',
+            'nlabel' => 'appinsights.requests.duration.milliseconds',
+            'unit'   => 'ms',
+            'min'    => '0',
+            'max'    => ''
+        },
+        'requests/failed' => {
+            'output' => 'Failed requests',
+            'label'  => 'requests-failed',
+            'nlabel' => 'appinsights.requests.failed.count',
+            'unit'   => '',
+            'min'    => '0',
+            'max'    => ''
+        },
+        'requests/rate' => {
+            'output' => 'Server request rate',
+            'label'  => 'requests-server-rate',
+            'nlabel' => 'appinsights.requests.server.perseconds',
+            'unit'   => 'requests/s',
+            'min'    => '0',
+            'max'    => ''
+        },
+        'performancecounters/requestExecutionTime' => {
+            'output' => 'HTTP request execution time',
+            'label'  => 'requests-execution-time',
+            'nlabel' => 'appinsights.requests.execution.time.milliseconds',
+            'unit'   => 'ms',
+            'min'    => '0',
+            'max'    => ''
+        },
+        'performancecounters/requestsInQueue' => {
+            'output' => 'HTTP requests in application queue',
+            'label'  => 'requests-queue',
```

Comment:
Maybe queued-requests would be better. @thibaults-centreon, what do you think?

Reply:
maybe keep the order to be consistent. But requests-queued or requests-in-queue.